### PR TITLE
[Term Entry] Variables: Add new term entry for variables

### PR DESCRIPTION
**Pull request by imfabra**
### Description

Added a new term entry for Dart variables in the path docs/content/dart/concepts/variables/variables.md. This includes a description of the term, syntax section, and an example explaining how variables work in Dart.

### Type of Change
- Adding a new entry

### Checklist
- [x] Checked for typos and formatting issues
- [x] Verified that the new entry follows documentation guidelines
- [x] Linked the PR to the related issue (#4237)
- [x] Assigned the PR to the appropriate reviewer

Please review the changes to ensure accuracy and completeness in the new term entry for Dart variables. Thank you!

### CODE REVIEW

File: content/dart/concepts/variables/variables.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+Tags:
+  - 'Variables'
+  - 'Integers'
+  - 'Booleans'
+  - 'Strings'
+CatalogContent:
+  - 'intro-to-flutter'
+---
+
+In Dart, **variables** are storage blocks that contain modifiable information during the execution of a program. They are essential for software development, as they allow storing, manipulating, and referencing data. We will explore the basics of variables in Dart, including their declaration, syntax, and practical examples.
```
Comment (ishg-153) [inline content/dart/concepts/variables/variables.md:15]:
```suggestion
In Dart, **variables** are storage blocks that contain modifiable information during the execution of a program. They are essential for storing, manipulating, and referencing data.
```

File: content/dart/concepts/variables/variables.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+### Print Variables
+
+To display the value of a variable in the console, you can use the print() function along with the variable name.
+
+```dart
+print("Name: $name");
+print("Age: $age");
+print("Height: $height");
+print("Is student? $isStudent");
+```
```
Comment (ishg-153) [inline content/dart/concepts/variables/variables.md:99]: This doesn't feel necessary in this context, please remove it.

**Review by ishg-153 — CHANGES_REQUESTED** [review]
@imfabra I noticed that through out the entry you have used [personal pronouns like you, we, etc. Please restructure the particular sentences to remove these personal pronouns as we don't use them in entries.

I have also suggested a couple of more changes, please let me know when you are done making them.

**Comment by ishg-153** [comment]
@imfabra It's been 2 weeks. Are you still working on the suggested changes?

**Comment by imfabra** [comment]
Hello @ishg-153 ,
Yes, I'm still working on the suggestions. I'll be finishing up in the next few hours.

**Comment by ishg-153** [comment]
@imfabra Due to lack of response from your end I am closing this PR and unassigning the corresponding issue.

**Pull request closed without merging**